db/deleteItem.go

```diff
@@ -0,0 +1,37 @@
+package db
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+)
+
+func (d *dynamoDB) DeleteItem(listID string, itemID string) error {
+	tableName := d.conf.TableNames.Items
+	if len(tableName) == 0 {
+		panic("Items table name not set")
+	}
+
+	input := &dynamodb.DeleteItemInput{
+		Key: map[string]*dynamodb.AttributeValue{
+			"ListId": {S: &listID},
+			"Id":     {S: &itemID},
+		},
+		TableName: aws.String(d.conf.TableNames.Items),
```

Comment:
You've looked this up and set it to `tableName` on line 10. Might as well use `tableName` here

Reply:
yep sorry, this was a copy and paste error